packages/system-data/src/fields/index.ts

```diff
@@ -28,9 +28,20 @@ import userFields from './users.yaml';
 import versionFields from './versions.yaml';
 import webhookFields from './webhooks.yaml';
 
-import type { FieldMeta } from '../types.js';
+import type { FieldIndex, FieldMeta } from '../types.js';
 
 export const systemFieldRows: FieldMeta[] = [];
+export const systemIndexRows: FieldIndex[] = [];
+
+export function isSystemField(collection: string, field: string): boolean {
```

Comment:
This is a duplicate function to the one on L35/46 and is causing the build to fail